src/main/resources/reference.conf

```diff
@@ -117,4 +120,10 @@ videobridge {
     # such candidates will be ignored.
     resolve-remote-candidates = false
   }
+
+  dtls {
```

Comment:
I'm wondering if, at least for the non-octo queue, we should scope this under something else rather than `dtls`, as the fact that it happens to live in the DTLS transport is implementation specific.  It's not really a "DTLS" queue, but really the main "ingress" queue.  Also, what about the sender-side queue size?

Reply:
Sorry, I've noticed now that this is actually the `egress` side, but I still have the same thought about the scoping of the parameter.

Reply:
yeah, that sounds like a good idea and I was wondering about this myself. Any suggestions?

Reply:
maybe:
```
transport {
  send {
    queue-size
  }
}
```
?

Reply:
done

Reply:
@bbaldino shall the element be named `transport` or `dtls-transport` ?

Reply:
I think just transport